ts.js

```diff
@@ -1,12 +1,26 @@
-module.exports = {
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
-  extends: [
-    './eslint.js',
-    'plugin:@typescript-eslint/recommended'
-  ],
+import tseslint from 'typescript-eslint';
+import nodePlugin from 'eslint-plugin-n';
+import eslint from './eslint.js';
+
+const nodeConfig = {
   rules: {
     'n/no-unsupported-features/es-syntax': ['error', { 'ignores': ['modules'] }],
     'n/no-missing-import': ['error', { 'tryExtensions': ['.js', '.ts'] }]
-  }
-};
+  },
+}
+
+const tslintConfig = tseslint.config({
+  languageOptions: {
+    parser: tseslint.parser,
```

Comment:
the tslintconfig will not work on ts

![ts](https://github.com/user-attachments/assets/c5087225-bf8a-4fad-a6d4-ddd7fd25bd5f)
![ts](https://github.com/user-attachments/assets/00ca9384-4aa5-43c0-b12b-db96f3f248a0)